backend/src/main/java/com/bakdata/conquery/apiv1/QueryProcessor.java

```diff
@@ -47,8 +53,7 @@
 	 * Creates a query for all datasets, then submits it for execution on the
 	 * intended dataset.
 	 */
-	public FullExecutionStatus postQuery(Dataset dataset, QueryDescription query, UriBuilder urlb, User user) {
-		user.authorize(dataset, Ability.READ);
+	public FullExecutionStatus postQuery(Dataset dataset, QueryDescription query, UriBuilder urlb, User user, boolean allProviders) {
```

Comment:
Würde es nicht sinn machen eher das FullExecutionStatus  erzeugen zu entkoppeln und man gibt hier nur noch die Execution raus? Dann kannst du dir viel klimbim in der Method sparen und musst die Methode nicht weiter anpassen für noch mehr zeug für die getStatus

Reply:
Der Status ist aber unterschiedlich bei den Executions, ganz entkoppeln geht hier nicht

Reply:
hast du aber hinbekommen :D